packages/manager/modules/nasha/src/dashboard/partition/accesses/accesses.template.html

```diff
@@ -0,0 +1,111 @@
+                    data-name="ip"
+                    data-items="$ctrl.authorizedAccesses"
+                    data-match="ip"
+                    data-group-by="'type'"
+                    data-model="$ctrl.model.ip"
+                    data-searchable
+                    data-required
+                >
+                    <span data-ng-bind="$item.ip"></span>
+                </oui-select>
+            </oui-datagrid-column>
+            <oui-datagrid-column
+                data-property="type"
+                data-title=":: ('nasha_dashboard_partition_accesses_list_type' | translate)"
+                data-type="options"
+                data-type-options="$ctrl.datagridTypeOptions"
+                data-filterable
+                data-sortable
+            >
+                <span
+                    data-ng-if="!$row.isForm"
+                    data-translate="{{ 'nasha_dashboard_partition_accesses_list_type_' + $row.type }}"
+                ></span>
+                <oui-select
+                    data-ng-if="$row.isForm"
+                    id="type"
+                    name="type"
+                    data-model="$ctrl.model.type"
+                    data-items="$ctrl.typeOptions"
+                    data-match="label"
+                    data-required
+                >
+                    <span data-ng-bind="$item.label"></span>
+                </oui-select>
+            </oui-datagrid-column>
+            <oui-datagrid-column>
+                <div
+                    data-ng-if="$row.isForm"
+                    class="d-flex justify-content-end align-items-center"
+                >
+                    <oui-spinner
+                        data-ng-if="$ctrl.isCreatingAccess"
+                        data-size="s"
+                        class="mr-2"
+                    ></oui-spinner>
+                    <oui-button
+                        class="mr-2"
+                        data-type="submit"
+                        data-disabled="!$ctrl.canCreateAccess"
+                    >
+                        <span class="oui-icon oui-icon-check"></span>
+                    </oui-button>
+                    <oui-button
+                        data-on-click="$ctrl.hideAccessForm()"
+                        data-disabled="$ctrl.isCreatingAccess || $ctrl.accesses.length === 1"
+                    >
+                        <span class="oui-icon oui-icon-close"></span>
+                    </oui-button>
+                </div>
+                <div
+                    data-ng-if="!$row.isForm"
+                    class="d-flex justify-content-end align-items-center"
+                >
+                    <oui-button data-on-click="$ctrl.goToDelete($row.ip)">
+                        <span class="oui-icon oui-icon-bin"></span>
+                    </oui-button>
+                </div>
+            </oui-datagrid-column>
+        </oui-datagrid>
+    </form>
+    <div data-ui-view></div>
```

Comment:
It is planned to add things below the datagrid ?

Reply:
Yes there are substates like the deletion state and the task tracking state

Reply:
We want the datagrid to remain displayed while substates like the one mentioned above are hit as they are modals
If the ui-view is placed on the datagrid's parent, the view will be emptied